compose-test.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+if [[ -z "${TRAVIS_PULL_REQUEST_BRANCH}" ]]; then
+  echo "BRANCH_NAME=${TRAVIS_BRANCH}" > .env
+else
+  echo "BRANCH_NAME=${TRAVIS_PULL_REQUEST_BRANCH}" > .env
+fi
+docker-compose -f api-tests/docker-compose.yml up -d || exit 1
```

Comment:
All the sanity checks
```bash
command || exit 1
```
can be avoided with the `-e` flag, which exits when a command fails:
```bash
#!/usr/bin/env bash
set -e
```